File: dotnet-docs-viewer/Common/Error/ErrorInfo.cs
using System.Text.Json.Serialization;

namespace WebApp.Common.Error
{
    public record ErrorInfo
    {
        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        public object? Unsupported { get; init; }

        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        public object? Supported { get; init; }

        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        public object? Errors { get; init; }

        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        public string? Errcode { get; init; }

        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        public string? Message { get; init; }

        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        public string? ExceptionName { get; init; }
    }
}

File: spring-docs-viewer-libre-office/jpa/src/main/java/com/docs/viewer/global/error/dto/ErrorInfo.java
package com.docs.viewer.global.error.dto;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
@JsonInclude(JsonInclude.Include.NON_NULL)
public class ErrorInfo {

    private Object unsupported;
    private Object supported;
    private Object errors;
    private String errcode;
    private String message;
    private String exceptionName;
}
